### PR TITLE
Add export statement for personal wallet

### LINKED ISSUE #735: Personal wallet statement export

### PR DESCRIPTION
**Pull request by mikozet**
Currently build on:
https://preview-dev.circles.garden/activities

### CODE REVIEW

**Review by juanenrisley — DISMISSED** [review]
👍🏿

**Review by louilinn — CHANGES_REQUESTED** [review]
We need to add a description for UBI payouts. I suggest writing `CirclesUBI` in column ` To_or_from_username` and `UBI payout` in `Payment_note` for those transactions. under condition safeAddress === ZERO_ADDRESS

Screenshot from 2023-09-25 15-44-48

**Review by louilinn — DISMISSED** [review]
Fixed :)
Merge pending for Lucia's acceptance

File: src/utils/fileExports.js [hunk 1]
```diff
@@ -113,8 +114,8 @@ const formatTransactions = async (transactions, safeAddress) => {
   return transactionData.map((data, index) => {
     if (data.otherSafe === ZERO_ADDRESS) {
       // UBI transaction
-      data.name = 'CirclesUBI';
-      data.paymentNote = 'UBI payout';
+      data.name = translate('ExportStatement.exportDataName');
+      data.paymentNote = translate('ExportStatement.exportPaymentNote');
```
Comment (louilinn) [inline src/utils/fileExports.js]: now we are adding some text in translations but not all. Maybe then I should add all fields in en.json

File: src/utils/fileExports.js [hunk 2]
```diff
@@ -203,7 +204,7 @@ export async function downloadCsvStatement(
 ) {
   // Verify date order
   if (startDate > endDate) {
-    throw new Error('Invalid date interval');
+    throw new Error(translate('ExportStatement.exportDataName'));
```
Comment (louilinn) [inline src/utils/fileExports.js]: this was the wrong text so I changed it in a commit

**Review by juanenrisley — APPROVED** [review]
👍🏿

**Pull request merged**